app/assets/stylesheets/components/_grid.sass

```diff
@@ -0,0 +1,12 @@
+$Grid-margin-base: $Theme-spacing-base
+
+.Grid
+  margin-bottom: $Grid-margin-base
+  margin-left: -$Grid-margin-base
+  margin-top: $Grid-margin-base
```

Comment:
The same issue I raised regarding margin-left applies here as well. Please inspect what this produces and confirm it is what you want.
You either make it negative, which is an internal style rather than an external (which would go against "**Most components should not set their own width, margin, and positioning**"), or simply remove this.